### PR TITLE
Hkparekh adds mountain view hometown file

### PR DESCRIPTION
**Pull request by hkparekh**
This pull request will add content to mountain view hometown file 🌞 

Some open ❓  before I merge:

1. Have you been to mountain view
1. Have you seen traffic near google office?
1. Which restaurant do you like?
1. @keran4 seems like we're located near each other

### CODE REVIEW

File: hkparekh-MountainView.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### MountainView, CA
+
+### Restaurants
+
+Shiva's
+Amarind Thai
+
+### For nature Lovers
+
+Shoreline Park
```
Comment (keran4) [inline hkparekh-MountainView.md]: I remember this!

**Comment by angpaul2** [comment]
@keran4 planning to come to mtv soon.heard its great there.

**Comment by hkparekh** [comment]
welcome back @keran4 See you in 2 hours

**Comment by hkparekh** [comment]
@keran4 - I addressed your feedback in commit c37ca8d

**Pull request merged**